src/connection/connection_sr.c

```diff
@@ -1161,8 +1163,11 @@ css_connect_to_master_server (int master_port_id, const char *server_name, int n
 
       //on newer systems (e.g. fedora 25) the following line of code
       //produces this warning: the use of `tempnam' is dangerous, better use `mkstemp'
+      auto[filename, fileptr] = filesys::open_temp_file ("cubrid");
+      pname = new char[filename.size () + 1];
+      std::copy (filename.begin (), filename.end (), pname);
+      pname[filename.size ()] = '\0';
```

Comment:
- Comment becomes outdated
- pname is freed by free() so it must be allocated by malloc. even better, you can use [strdup ](https://en.cppreference.com/w/c/experimental/dynamic/strdup)function.
```suggestion
      auto[filename, fileptr] = filesys::open_temp_file ("cubrid");
      pname = new char[filename.size () + 1];
      std::copy (filename.begin (), filename.end (), pname);
      pname[filename.size ()] = '\0';
```

Same comment applies to connection_cl.c change too.